packages/auth/src/mergeAccessPolicies.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+/**
+ * Merge policy1 and policy2 together to form a superset access policy
+ * @param {AccessPolicyObject} policy1
+ * @param {AccessPolicyObject} policy2
+ * @returns {AccessPolicyObject} mergedPolicy
+ */
+export const mergeAccessPolicies = (policy1, policy2) => {
```

Comment:
This function _**only**_ works with the modern `AccessPolicyObject` format (rather than the legacy one with permission groups). I don't actually know why and when the legacy access policies are being created... Perhaps some error handling here is required...